### DISCUSSION TITLE
Can't See Penumbra in XL

### DISCUSSION THREAD

**Discussion by Azraenya** [xivdev/Penumbra#428] [General]
I'm sure this is a very stupid question, and I'm sorry for that!  However, when I'm following instructions from https://reniguide.carrd.co/, I put in the repo linked in that guide.  According to the guide, it seems that I should see Penumbra as a plugin already, but it's not there when I search for it.  I then put in the repo link for Penumbra itself as shown in the installation guide on the main page here, saved changes, and again looked for Penumbra, but it's still not showing up.  Note that I am fairly new to using XIVLauncher, but a few days ago, Penumbra did show up in the Dalamud plugin list, saying it was outdated and incompatible.  It seems like since the update for Penumbra, I can't get it to show up at all.  I've deleted all XIVLauncher info and reinstalled it, all with the same problem.  Any advice?  Sorry to be a bother!

**Comment by SerenitySublime** [comment]
Hi. Reni here. Penumbra is still in testing and the instructions I provide are only for the stable version of Penumbra. Please wait until Penumbra is released as a stable version and then you can download it at that time. Testing is not early access.
